packages/account-v2/src/containers/VerificsationStatus/VerificationStatus.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { DerivLightApprovedPoiIcon, DerivLightWaitingPoiIcon } from '@deriv/quill-icons';
+import { Button, Text } from '@deriv-com/ui';
+import { IconWithMessage } from '../../components/IconWithMessage';
+import { ACCOUNT_V2_ROUTES, AUTH_STATUS_CODES, POI_SERVICE } from '../../constants';
+import { TPOIStatus, TSupportedPOIServices } from '../../types';
+
+type TVerificationStatusProps = {
+    isPOARequired?: boolean;
+    service: TSupportedPOIServices;
+    status: TPOIStatus;
+};
+
+const ContinueToPOA = () => {
```

Comment:
wouldn't it be better to create one component with two props: path and buttonText?

Reply:
Will make the change